api/src/main/java/org/openmrs/module/kenyaemr/reporting/library/shared/common/CommonCohortLibrary.java

```diff
@@ -197,7 +191,7 @@ public CohortDefinition transferredOut() {
 		cd.addParameter(new Parameter("onOrBefore", "Before Date", Date.class));
 		cd.addParameter(new Parameter("onOrAfter", "After Date", Date.class));
 		cd.setName("transferred out between dates");
-		cd.setTimeModifier(PatientSetService.TimeModifier.ANY);
+		//cd.setTimeModifier(PatientSetService.TimeModifier.ANY);
 		cd.setQuestion(reasonForDiscontinue);
 		cd.setOperator(SetComparator.IN);
```

Comment:
Let's restore this as well

Reply:
This was causing the failing reporting 0.0.9 =>cd.setTimeModifier(PatientSetService.TimeModifier.ANY); It should be restored